components/RomPatcherUi.vue

```diff
@@ -5,18 +5,26 @@
             <h3 class="patcher-header">{{ $t('chokuretsu-rom-patcher-options') }}</h3>
             <table id="patcher-options">
                 <tbody>
+                    <RomPatcherOptionDescription :title="$t('chokuretsu-rom-patcher-dub-title')" img="/images/chokuretsu/opening-subtitles.png"
```

Comment:
Obviously we will need a real image for this for the Italian release, but I don't know what to put lol